src/Timeline/ProfileManager.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace AppBundle\Timeline;
+
+use AppBundle\Algolia\ManualIndexerInterface;
+use AppBundle\Entity\Timeline\Profile;
+
+class ProfileManager
+{
+    private $algolia;
+
+    public function __construct(ManualIndexerInterface $algolia)
+    {
+        $this->algolia = $algolia;
+    }
+
+    public function postPersist(Profile $profile): void
```

Comment:
Can you add an `AlgoliaIndexedEntityInterface` so we have only one `AlgoliaIndexedEntityManager`?

Reply:
reminder: `MeasureManager::unIndex` method slightly differs from others

Reply:
Then in `AlgoliaIndexedEntityManager::index($entity)`, do `if ($entity instanceof Measure)`

Reply:
Or maybe add an `getElementsToIndex` method in the `AlgoliaIndexedEntityInterface`?

Reply:
i guess the functional logic is too specific.
`MeasureManager::unIndex` method needs to re-index related themes and un-index the measure.
when `MeasureManager::index` method needs to index related themes and index the measure.

I prefer your first option:
```
Then in AlgoliaIndexedEntityManager::index($entity), do if ($entity instanceof Measure)
```
:ok_hand: